uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -71,6 +74,28 @@ function hrcore_civicrm_summaryActions( &$actions, $contactID ) {
   $actions['otherActions']['user-add'] = $userAdd;
 }
 
+/**
+ * Implementation of hook_addContactMenuActions to add the
+ * Workflow menu group to the contact actions menu.
+ *
+ * @param ActionsMenu $menu
+ */
+function tasksassignments_addContactMenuActions(ActionsMenu $menu) {
```

Comment:
This is inside the hrcore extension, so it should be `hrcore_` instead of `taskassignments_`

Reply:
Oops, I copied it over from T&A and didn't change the name.